pkg/operation/botanist/component/etcd/etcd.go

```diff
@@ -616,6 +619,36 @@ func (e *etcd) SetOwnerCheckConfig(ownerCheckConfig *OwnerCheckConfig) {
 	e.ownerCheckConfig = ownerCheckConfig
 }
 
+func (e *etcd) ScaleETCD(ctx context.Context, replicas int) error {
```

Comment:
Nit Nit: Just call it `Scale` since the package already suggests that it's about etcd :)

Reply:
Nit Nit fixed 😄